package.json

```diff
@@ -16,7 +16,7 @@
     "build-docs": "yarn --cwd www run build",
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s",
     "start": "yarn --cwd www run develop",
-    "ci-lint": "eslint --rule \"prettier/prettier: 2\"",
+    "ci-lint": "eslint",
```

Comment:
should drop this entirely and just use the normal eslint command if we're doing this